index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+  <head>
+
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Chris Closser</title>
+    <!-- <link rel="stylesheet" href="vendor/styles/reset.css"> -->
+    <link rel="stylesheet" href="styles/base.css">
+    <script type="text/javascript" scr="reset.css">
+
+    </script>
+  </head>
+  <body>
+    <header class="header">
+      <div class="my-name">
+        Chris Closser
+      </div>
+      <button class="burger">&#9776;</button>
+      <nav class="hidden" id="main-nav">
+        <ul>
+          <li><a href="#">Home</a></li>
+          <li><a href="https://github.com/ChristopherSClosser">GitHub</a></li>
+          <li><a href="#">Projects</a></li>
+          <li><a href="#">Gallery</a></li>
+          <li><a href="#">Me</a></li>
+        </ul>
+      </nav>
+    </header>
+    <div class="name">
```

Comment:
Could you wrap these div(soon to be section :) tags in a main tag to describe that it is the main body of your code?